src/rng.h

```diff
@@ -196,4 +196,13 @@ std::optional<tripoint> random_point( const tripoint_range<tripoint> &range,
 std::optional<tripoint> random_point( const map &m,
                                       const std::function<bool( const tripoint & )> &predicate );
 
+/**
+ * Round a value with the fractional part as the probability of rounding away from zero
+ * e.g. 4.2 rounds to 4 20% of the time or to 5 80% of the time
+ *
+ * @param val Value to be rounded.
+ * @return Rounded value.
+ */
+double round_rand( double val );
```

Comment:
may be just me, but this name seems not really descriptive of what it actually does - round away from 0 is likely not what most would expect from innocent named "round" function, it's also dissimilar (in name style) to almost identical roll_remainder

Reply:
I spent at least five minutes searching through the existing rnd functions and failed to find roll_remainder, which I agree is almost identical. I'll switch to using that one.